tests_lib/helpers/web_ui/base_page.py

```diff
@@ -1,9 +1,10 @@
 from selenium import webdriver
 from selenium.webdriver.support import expected_conditions as ec
 from selenium.webdriver.support.wait import WebDriverWait
+from abc import ABC, abstractmethod
 
 
-class BasePage:
+class BasePage(ABC):

```

Comment:
The Docstring to the constructor __init__ is confusing.
I suggest changing "Initialize the BasePage with a WebDriver instance"  to "Shared constructor for base classes with a WebDriver instance. BasePage is an ABC, so the instance can't be created directly."